docs/docs/installation/index.md

```diff
@@ -94,6 +94,37 @@ Configuration is updated at ~/.dstack/config.yml
 
 This configuration is stored in `~/.dstack/config.yml`.
 
+### (Optional) CLI Autocompletion
+
+`dstack` supports shell autocompletion for `bash` and `zsh`.
+
+=== "bash"
+    
+    <div class="termy">
+    
+    ```shell
+    $ eval "$(dstack completion bash)"
+    $ echo "$(dstack completion bash)" >> ~/.bashrc
```

Comment:
I'm not sure it's a good idea to write directly to ~/.bashrc or ~/.zshrc. Let me fix it.